packages/ctt/__tests__/ctc.test.ts

```diff
@@ -0,0 +1,94 @@
+import { synthSnapshot } from 'projen/lib/util/synth';
+import { describe, expect, it } from 'vitest';
+
+import { CalmsTypescriptCdk } from '../src';
+
+const requiredFileNames = [
+  '.commitlintrc.json',
+  '.gitattributes',
+  '.github/pull_request_template.md',
+  '.github/workflows/auto-queue.yml',
+  '.github/workflows/build.yml',
+  '.github/workflows/copilot-setup-steps.yml',
+  '.github/workflows/pull-request-lint.yml',
+  '.github/workflows/release.yml',
+  '.github/workflows/update-snapshots.yml',
+  '.github/workflows/upgrade-main.yml',
+  '.gitignore',
+  '.husky/commit-msg',
+  '.husky/pre-commit',
+  '.husky/pre-push',
+  '.lintstagedrc.json',
+  '.mergify.yml',
+  '.npmrc',
+  '.prettierignore',
+  '.prettierrc.json',
+  '.projen/deps.json',
+  '.projen/files.json',
+  '.projen/tasks.json',
+  'cdk.json',
+  'eslint.config.mjs',
+  'LICENSE',
+  'package.json',
+  'README.md',
+  'tsconfig.json',
+  'vitest.config.mts',
+];
```

Comment:
Where it doesn't affect functionality always order arrays in alphabetical order. Add that as a repo-level instruction.